instrumentation/dubbo/dubbo-apache-2.7/javaagent/src/test/groovy/io/opentelemetry/instrumentation/dubbo/apache/v2_7/DubboTest.groovy

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.dubbo.apache.v2_7
+
+import io.opentelemetry.instrumentation.dubbo.apache.v2_7.api.HelloService
+import io.opentelemetry.instrumentation.dubbo.apache.v2_7.impl.HelloServiceImpl
+import io.opentelemetry.instrumentation.test.AgentTestTrait
+import org.apache.dubbo.config.ReferenceConfig
+import org.apache.dubbo.config.RegistryConfig
+import org.apache.dubbo.config.ServiceConfig
+
+class DubboTest extends AbstractDubboTest implements AgentTestTrait {
+  @Override
+  ServiceConfig configureServer() {
```

Comment:
We generally have the tests differ only in the way they are instrumenting. Is it possible to have the registry / service definition happen in the base class, only difference is non-agent version calls addFilter if it's required?

Reply:
yes,I will fix it.

Reply:
I've made the unit test code cleaner